.github/workflows/pr-vrt-poc-comment.yml

```diff
@@ -56,4 +64,4 @@ jobs:
           message: |
             VR Regression Report:
             - status: ${{ github.event.workflow_run.conclusion }}
-            - check whole [report](${{github.event.workflow_run.workflow_url}}) to review visual differences
+            - check whole [report](${{github.event.workflow_run.html_url}}) to review visual differences
```

Comment:
I am not sure, but in order to lead to desired page maybe something like this would work, wdyt? 

` - check whole [report](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.event.workflow_run.id }}) to review visual differences
`